### PR TITLE
Fix websockets

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,10 @@ Please check it reguarly.
 The following instructions take you through getting the most basic setup working.
 We'll explain what actually happens below the hood, and how to configure things afterwards.
 
+### Update hosts file
+
+As the website is expected to run at `local.exercism.io`, you should update your hosts file to have `local.exercism.io` resolve to `127.0.0.1` (localhost).
```
Comment (iHiD) [inline README.md:31]: This should actually be supported automatically by the DNS. But I might have configured it wrong. I'm happy for this to go in here for now either way.

File: stack.default.yml [hunk 2]
```diff
@@ -17,16 +17,16 @@ configure:
   # setup:
   #   build:
   #     args:
-  #       exercism_config_version: 0.56.0
+  #       exercism_config_version: 0.60.0
```
Comment (iHiD) [inline stack.default.yml:20]: I might be wrong, but I believe that the setup gem is build using the version in the `Gemfile.lock` of this repo if this is left commented out. So we should also run `bundle update` as part of this PR to keep that comment in sync with the `Gemfile.lock`.

File: README.md (hunk 1 above)
Comment (ErikSchierboom) [reply README.md:31]: It is? I can check that in a sec

File: stack.default.yml (hunk 2 above)
Comment (ErikSchierboom) [reply stack.default.yml:20]: Ah, I'll update! I'm always slightly confused about the setup bit as that was developed in my holiday :)

File: README.md (hunk 1 above)
Comment (ErikSchierboom) [reply README.md:31]: It doesn't work for me without the hosts file update.

File: stack.default.yml (hunk 2 above)
Comment (ErikSchierboom) [reply stack.default.yml:20]: I've updated.

File: README.md (hunk 1 above)
Comment (iHiD) [reply README.md:31]: Annoying! :)

**Pull request merged**